docs/_posts/ahmet-mesut/2024-02-13-bert_sequence_classifier_biomarker_en.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: model
+title: Bert For Sequence Classification (Biomarker)
+author: John Snow Labs
+name: bert_sequence_classifier_biomarker
+date: 2024-02-13
+tags: [en, licensed, bfsc, biomarker, classification]
+task: Text Classification
+language: en
+edition: Healthcare NLP 5.2.1
+spark_version: 3.0
+supported: true
+engine: tensorflow
+annotator: MedicalBertForSequenceClassification
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is a [BioBERT](https://nlp.johnsnowlabs.com/2022/07/18/biobert_pubmed_base_cased_v1.2_en_3_0.html) based sentence classification model that can determine whether the clinical sentences include terms related to biomarkers or not.
```

Comment:
Is this link correct? You didn't use this embeddings as far as I know.

Reply:
fixed FYI